packages/ui/src/theme/__tests__/themeType.test.ts

```diff
@@ -0,0 +1,63 @@
+import { Theme } from '../index';
+
+describe('Amplify UI Theme', () => {
+  describe('components', () => {
+    it('should allow empty components', () => {
+      const theme: Theme = {
+        name: 'test',
+        tokens: {
+          components: {
+            alert: {},
```

Comment:
Is there a better way to test a type shape like this? Also, if we add a component we will need to add it here. Grabbing all the component names and creating an object dynamically will not trigger a TS error.